Automattic/jetpack
# Masterbar: Add menu structure

This should be functionally ready for testing.

#### Changes proposed in this Pull Request:

* Adds menu structure for unified admin menu.
* Adds unit tests for each new menu item and some utility methods

#### Jetpack product discussion


pbAPfg-VL-p2#comment-1904

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






To test locally:
* Add a mu-plugin that contains: ` add_filter( 'jetpack_load_admin_menu_class', '__return_true' );`
* Make sure your site is connected to wp.com and has the masterbar module activated.
* Load wp-admin and check the sidebar!

To test with an Atomic site:
* [Install & activate the Jetpack Beta Plugin](https://jetpack.com/download-jetpack-beta/) on your Atomic test site.
* Go to Jetpack > Jetpack Beta
* Under "Search for Jetpack Feature Branch" enter this branch or PR id: 17629 => Click Activate
* Add a mu-plugin that contains: ` add_filter( 'jetpack_load_admin_menu_class', '__return_true' );`
* Look at the admin menu and admire the new Menu structure.
* Click on a menu item that takes you to Calypso and marvel at this new achievement.


#### Proposed changelog entry for your changes:

* Not sure yet.
